docs/en_US/NAS/SPOS.rst

```diff
@@ -106,6 +100,7 @@ Known Limitations
 
 
 * Block search only. Channel search is not supported yet.
+* Weight inheriting while search is not supported yet.
```

Comment:
In the search phase, training from scratch is required. Inheriting weights from super-net is not supported yet.

Reply:
I think it would be "training from the pretrained checkpoint"?